app/templates/todos/index.plush.html

```diff
@@ -18,28 +18,28 @@
         </thead>
         <tbody>
         <%= for (todo) in todos { %>
-        <tr>
-            <td class="text-center" scope="row">
-                <i class="text-muted far fa-check-circle"></i>
-            </td>
-            <td>
-                <a class="todos-links-color" href="#">
-                    <%= todo.Title %>
-                </a>
-            </td>
-            <td>
-                <%= todo.LimitDate.Format("Monday 02, January 2006") %>
-            </td>
-            <td>
-                <div class="row mx-0 justify-content-center">
-                    <i class="text-muted fas fa-pencil-alt mx-3"></i>
-                    <a class="delete-item" href="#" data-entity="ToDo" data-url="<%= deleteTodoPath({todo_id: todo.ID}) %>" data-toggle="modal"
-                       data-target="#deleteModal">
-                        <i class="text-danger far fa-trash-alt mx-3"></i>
+            <tr>
+                <td class="text-center" scope="row">
+                    <i class="text-muted far fa-check-circle"></i>
+                </td>
+                <td>
+                    <a class="todos-links-color" href="<%= editTodoPath({todo_id: todo.ID}) %>">
+                        <%= todo.Title %>
                     </a>
-                </div>
-            </td>
-        </tr>
+                </td>
+                <td>
+                    <%= todo.LimitDate.Format("Monday 02, January 2006") %>
+                </td>
+                <td>
+                    <div class="row mx-0 justify-content-center">
+                        <i class="text-muted fas fa-pencil-alt mx-3"></i>
```

Comment:
This one should point to the edit form of the todo

Reply:
Fixed